tidb-5.0-a-one-stop-htap-database-solution.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 'TiDB 5.0: a One-Stop HTAP Database Solution'
+author: ['Shawn Ma']
+date: 2021-06-03
+summary: TiDB 5.0 features a comprehensive HTAP solution enhanced by an MPP analytical query engine. This article introduces the new TiDB 5.0 HTAP architecture and how TiDB 5.0 serves various transactional & analytical hybrid workload scenarios.
+tags: ['HTAP', 'TiDB 5.0', 'Real-time analytics']
+categories: ['Product']
+image: /images/blog/tidb-5.0-a-one-stop-htap-database-solution.jpg
+---
+
+**Author:** [Shawn Ma](https://github.com/ilovesoup) (Tech Lead of the Real-time Analytics team at PingCAP)
+
+**Transcreator:** [Ran Huang](https://github.com/ran-huang); **Editor:** Tom Dewan
+
+![TiDB 5.0: a One-Stop HTAP Database Solution](media/tidb-5.0-a-one-stop-htap-database-solution.jpg)
```

Comment:
```suggestion
![TiDB 5.0: an open source SQL database for real-time anlytics](media/tidb-5.0-a-one-stop-htap-database-solution.jpg)
```

Reply:
For SEO

Reply:
Next time, you can include our SEO keywords in the image name. This helps improve our SEO ranking.